cefies/routes/auth.py

```diff
@@ -38,8 +49,9 @@ def register(data: RegisterData):
     new_user.email = data.email
     new_user.name = data.name
     new_user.password = get_password_hash(data.password)
-    # TODO: Upload file to object and set avatar
-    new_user.avatar = ""
+    avatar_content = await form.avatar.read()
+    avatar_url = bucket.upload_file(avatar_content, get_hash_sha256(avatar_content))
```

Comment:
Use `loop.run_in_executor()`. This is a blocking call, all other process will be blocked until upload is finished

https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.loop.run_in_executor

Reply:
Thank you Ren! I already fixed it.